src/TodoListItem.jsx

```diff
@@ -0,0 +1,8 @@
+const TodoListItem = ({ todo }) => <li>{todo.title}</li>;
+
+// I am getting these eslint errors here: 
+// 'todo' is missing in props validation eslint react/prop-types
+// 'todo.title' is missing in props validation eslint react/prop-types
+// so, I added 'react/prop-types': 'off' to eslint rules, is there a better solution? 
```

Comment:
This is great, let's keep it off till we reach the stage where prop types are required